.npmignore

```diff
@@ -0,0 +1,5 @@
+dist/demo/**
```

Comment:
not needed, we are going to have a `prepare` script which calls the clean so there won't be any extra files in the `dist` folder. See #694